contrib/redhat/collectd.spec

```diff
@@ -823,11 +837,15 @@ Monitors process starts/stops via netlink library.
 Summary:	Python plugin for collectd
 Group:		System Environment/Daemons
 Requires:	%{name}%{?_isa} = %{version}-%{release}
-	%if 0%{?rhel} && 0%{?rhel} < 6
+%if 0%{?rhel} && 0%{?rhel} > 7
```

Comment:
this will also catch rhel8, but there is no python2.

Reply:
are you saying python2 is only in centos?

Reply:
python2 is in centos 7 (and RHEL7), python3 is in centos 8 and RHEL 8. There is no python2 in centos8 or RHEL8. The change will try to pull in python2 in cenots8 or rhel8.

Reply:
Sorry I thought you meant there was a difference between centos8 and rhel8. Odd I tested this worked on centos8. I'm checking asap thanks